packages/lib-classifier/src/plugins/tasks/dropdown-simple/README.md

```diff
@@ -24,7 +24,7 @@ If we need to enable the "Other" option in the future, please look at `component
 
 Simple Dropdown task data structure, example:
 
-```
+```javascript
 "T0":{
   "help":"",
   "type":"dropdown-simple",
```

Comment:
This code snippet is JSON, not JavaScript.

Reply:
Fixed!